libcloudforensics/providers/gcp/internal/log.py

```diff
@@ -26,7 +26,7 @@ class GoogleCloudLog:
   """Class representing a Google Cloud Logs interface.
 
   Attributes:
-    project_id: Google Cloud project ID.
+    project_ids: Comma-separated list of Google Cloud project IDs.
```

Comment:
I think it would be nicer to have this as a Python list, since these really are libraries that we're providing and the CLI is just a reference interface...the CLI function can do the `split(',')`